gslb/cache/controller_obj_cache.go

```diff
@@ -763,6 +851,21 @@ func PopulateHMCache(createSharedCache bool) *AviHmCache {
 	return aviHmCache
 }
 
+func PopulateSPCache(createSharedCache bool) *AviSpCache {
```

Comment:
Do you plan to call this function with other arguments ? Currently I see that we are only calling this once.

Reply:
No, not right now. Will remove the argument.